pkg/hwinfo/hardware.go

```diff
@@ -622,16 +623,25 @@ func NewHardwareDevice(hd *C.hd_t) (*HardwareDevice, error) {
 		ParentUdi:         C.GoString(hd.parent_udi),
 		Resources:         resources,
 		Detail:            detail,
-		Driver:            C.GoString(hd.driver),
-		DriverModule:      C.GoString(hd.driver_module),
-		Drivers:           ReadStringList(hd.drivers),
-		DriverModules:     ReadStringList(hd.driver_modules),
 		UsbGUID:           C.GoString(hd.usb_guid),
 		DriverInfo:        driverInfo,
 		ModuleAlias:       C.GoString(hd.modalias),
 		Label:             C.GoString(hd.label),
 	}
 
+	if ephemeral {
+		// loaded kernel modules are inherently stateful and including them by default will make the report unstable
```

Comment:
We depend on these drivers to be able to boot NixOS. And we have not other way to interfere them, hence I would always include them, even though it might change in theory across kernels, I don't see this happening in practice.

Reply:
Reverted this